src/pages/CubePlaytestPage.js

```diff
@@ -242,6 +242,7 @@ CustomDraftCard.propTypes = {
     title: PropTypes.string.isRequired,
     html: PropTypes.string,
     markdown: PropTypes.string,
+    defaultSeats: PropTypes.number,
```

Comment:
This needs to be updated both places that draft formats are used in the prop types.